snowpark-checkpoints-collectors/src/snowflake/snowpark_checkpoints_collector/summary_stats_collector.py

```diff
@@ -23,9 +23,13 @@
 
 from pyspark.sql import DataFrame as SparkDataFrame
 from pyspark.sql.functions import col
-from pyspark.sql.types import BooleanType, FloatType, IntegerType, StructField
+from pyspark.sql.types import BinaryType as SparkBinaryType
+from pyspark.sql.types import BooleanType, IntegerType, StructField
+from pyspark.sql.types import DateType as SparkDateType
 from pyspark.sql.types import DoubleType as SparkDoubleType
+from pyspark.sql.types import FloatType as SparkFloatType
 from pyspark.sql.types import StringType as SparkStringType
+from pyspark.sql.types import TimestampType as SparkTimestampType
```

Comment:
I think this can use a simplifier version such as:
```suggestion
from pyspark.sql.types import (
    BinaryType as SparkBinaryType,
    BooleanType,
    IntegerType,
    StructField,
    DateType as SparkDateType,
    DoubleType as SparkDoubleType,
    FloatType as SparkFloatType,
    StringType as SparkStringType,
    TimestampType as SparkTimestampType,
)
```

Reply:
Agree! On going

Reply:
It's ok